src/api/api.js

```diff
@@ -0,0 +1,51 @@
+export async function getUserInfo() {
+  const response = await fetch(`${process.env.REACT_APP_BASE_URL}/sample/user`);
```

Comment:
api 함수를 작성할 때 마다 `process.env.REACT_APP_BASE_URL` 구문을 계속 넣어줘야 하는데요~
이렇게 사용할 수 있도록 만들어본다면 어떨까요?
```suggestion
  const response = await http.get('/sample/user');
```

Reply:
axios와 같은 라이브러리를 사용한다면 더욱 편하게 만들어볼 수 있어요~